### DISCUSSION TITLE
NavigationView Language

### DISCUSSION THREAD

**Discussion by nevelate** [amwx/FluentAvalonia#527] [Q&A] [answered]
I have NavigationView and my system language is Russian. NavigationView Settings and all tooltips are russian now. How I can change app language?
![image](https://github.com/amwx/FluentAvalonia/assets/59573371/389c42bd-2278-4946-bd1f-bf5ff74cdd21)

**Comment by sccombp** [comment, marked as answer]
If you just need to change the language for the application you can set the culture for the current thread (by default it is set to your system language). You can do this in your App.axaml.cs file.

```csharp
    public override void OnFrameworkInitializationCompleted()
    {
        CultureInfo ci = new CultureInfo("uk-UA");
        Thread.CurrentThread.CurrentCulture = ci;
        Thread.CurrentThread.CurrentUICulture = ci;

        if (ApplicationLifetime is IClassicDesktopStyleApplicationLifetime desktop)
        {
            desktop.MainWindow = new MainWindow
            {
                DataContext = new MainWindowViewModel(),
            };
        }

        base.OnFrameworkInitializationCompleted();
    }
```
